hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/web/ozShell/keys/ListKeyHandler.java

```diff
@@ -50,7 +50,8 @@
   private int maxKeys;
 
   @Option(names = {"--start", "-s"},
-      description = "The first key to start the listing")
+      description = "The key to start the listing from. " +
```

Comment:
To be consistent with `volume list` and `bucket list`, can you please replace the space between the two sentences with a newline?

```suggestion
      description = "The key to start the listing from.\n" +
```

For reference:

https://github.com/apache/hadoop-ozone/blob/8f3a42f35826aa59ceb2748bf64dca51e548a1bb/hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/web/ozShell/volume/ListVolumeHandler.java#L53-L56

https://github.com/apache/hadoop-ozone/blob/8f3a42f35826aa59ceb2748bf64dca51e548a1bb/hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/web/ozShell/bucket/ListBucketHandler.java#L53-L56